conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -32,7 +32,6 @@ install:
 
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda.exe update --yes --quiet conda
```

Comment:
FWIW this is a holdover from when we use to support Python 3.4. Given we don't do that any more and `conda` is generally recent enough, it seems reasonable to drop.